onig_sys/build.rs

```diff
@@ -37,6 +37,9 @@ impl fmt::Display for LinkType {
 }
 
 fn env_var_bool(name: &str) -> Option<bool> {
+    if !name.starts_with("CARGO") {
```

Comment:
I'm wondering if we should invert this and do something like 'if name.starts_with("RUSTONIG")` instead.